src/index.js

```diff
@@ -25,7 +25,8 @@ logger.info("🤖 Initializing middleware");
 app.use(morgan("tiny", { stream: logger.stream }));
 app.use(
   cors({
-    origin: `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`
+    origin: `http://localhost:3000`
+    // origin: `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`
```

Comment:
I took a look at [the `cors` docs](https://www.npmjs.com/package/cors#configuration-options), apparently `origin` can be an array! So you can do

```suggestion
origin: [`http://localhost:3000`,  `blah-blah.netlfiy.com`, `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`]
```

and they'll all be allowed!

Reply:
🤸‍♀️ Sweet, thanks!